src/common/GPOWrapper/pch.h

```diff
@@ -1,4 +1,4 @@
 ﻿#pragma once
-#include <unknwn.h>
+#include <Unknwn.h>
```

Comment:
This is that include file case thing I called out. This repository generally used the correct case (unlike Terminal which generally used the incorrect case), but both mixed cases, and an engineer for Terminal looked up the correct case and at least some of the story. Anyway, since I'm already changing things, it seemed worthwhile to offer this correction as well.

(There are definitely other files that have inconsistent include case, but I'm only touching one here -- I'd be inclined to make a PR specific to that for the rest, but that's for another day.)